### PR TITLE
Add `Learn Log4j in 5 minutes!` page

### LINKED ISSUE #2507: Create a `Learn Log4j in under 2 minutes!` page
As agreed in [the site structure document](https://docs.google.com/document/d/10Fu7oqDzdM_D6LbexzwX9arh51Tic7AGvkWTQrL6jjQ/edit), create a _"Learn Log4j in under 2 minutes!"_ page with following goals in mind:

- Outline:
  - `pom.xml` entries
  - `logger.info()` (and `logger.atInfo()`, i.e., the fluent API)
  - Don’t use `toString()`
  - Don’t `printStackTrace()` – use last argument
  - Don’t string concatenate – use parameters
  - My first `log4j2.xml`
  - Provide quick links to manual pages for next steps
- Target `2.x`, port to `main`

### PR DESCRIPTION
**Pull request by vy**
Fixes #2507.

### CODE REVIEW

File: src/site/antora/modules/ROOT/pages/5min.adoc [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+
+There are several widespread bad practices.
+Let's try to walk through the most common ones.
+
+[#pitfal-toString]
+==== Don't use `toString()`
+
+[source,java]
+----
+// [✗] `Object#toString()` is redundant in arguments
```
Comment (ppkarwasz) [inline src/site/antora/modules/ROOT/pages/5min.adoc:157]: I have the impression that users often don't read the explanation if it is contained in a source code comment. Maybe the rationale should also be written as text?

File: src/site/antora/modules/ROOT/pages/5min.adoc [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+  testOnly 'org.apache.logging.log4j:log4j-slf4j2-impl' // <1>
+
+}
+----
+====
+<1> Note that the logging implementation and bridges are only needed for tests!
+<2> SLF4J is another widely used logging API.
+`log4j-slf4j2-impl` forwards SLF4J calls to Log4j API, which effectively gets processed by Log4j Core too.
+
+Next, you need a `src/**test**/resources/log4j2.xml`.
```
Comment (ppkarwasz) [inline src/site/antora/modules/ROOT/pages/5min.adoc:388]:
```suggestion
Next, you need a `src/**test**/resources/log4j2-test.xml`.
```

**Review by ppkarwasz — APPROVED** [review]
LGTM

I have added some notes below.

File: src/site/antora/modules/ROOT/pages/5min.adoc [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+----
+
+This provides observability into an application's runtime. (See {logging-services-url}/what-is-logging.html[What is logging?] page for a longer read.)
+
+But we can do way better than a `printf()` statement!
+
+* Enhance the output with additional information (timestamp, class & method name, line number, host, severity, etc.)
+* Use different **layouts** (CSV, JSON, etc.)
+* Forward to different **appenders** (file, socket, database, queue, etc.)
+* **Filter** on demand (e.g., increase verbosity dynamically for troubleshooting purposes)
```
[the remark this answers is not in the document]
Comment (vy) [reply src/site/antora/modules/ROOT/pages/5min.adoc]: That is a great correction! Applied in fb1f06bdbd2ac958f35b0fcbcebc801240ec73ba.

File: src/site/antora/modules/ROOT/pages/5min.adoc [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+    public void truncateTable(String tableName) throws IOException {
+        LOGGER.warn("truncating table `{}`", tableName); // <2>
+        db.truncate(tableName);
+    }
+
+}
+----
+<1> This is a thread-safe, reusable `Logger` instance.
+The associated class will be captured at initialization – no need for a `getLogger(DbTableService.class)`.
+<2> The generated **log event** will be automatically parameter formatted (note the parameter placeholder, i.e., `{}`) and enriched with **level** (i.e., `WARN`), timestamp, class & method name, line number, and several other information.
```
[the remark this answers is not in the document]
Comment (vy) [reply src/site/antora/modules/ROOT/pages/5min.adoc]: Fixed in 280462133cd5ea8dec7edfdddbc8445519df0707.

File: src/site/antora/modules/ROOT/pages/5min.adoc (hunk 1 above)
Comment (vy) [reply src/site/antora/modules/ROOT/pages/5min.adoc]: Fixed in 34c325c6e4aa214c737c380af3a481c6b82a4f26. While doing so, learned something called [Antora checklists](https://docs.antora.org/antora/latest/asciidoc/checklists/).

File: src/site/antora/modules/ROOT/pages/5min.adoc (hunk 2 above)
Comment (vy) [reply src/site/antora/modules/ROOT/pages/5min.adoc]: Fixed in 89f8827f104be82b799f942d5dc4528fe93031c4.

**Pull request merged**